<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>update_stu_box</class>
 <widget class="QDialog" name="update_stu_box">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>524</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>714</width>
    <height>524</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>714</width>
    <height>524</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Update student</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background:#1F2937;
	color:white;
	font-weight:700;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>571</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
        font-size:35px;
        font-family:Arial, sans-serif;
  color:#ccc;
}</string>
   </property>
   <property name="text">
    <string>UPDATE STUDENT INFORMATION</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update_f_names_btn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
		font-family:Arial, sans-serif;
		font-size:18px;
        font-weight:800;
        border-radius:12px;
        background:#0C4A6E;
}</string>
   </property>
   <property name="text">
    <string>UPDATE FIRST NAME(S)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update_l_name_btn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
		font-family:Arial, sans-serif;
		font-size:18px;
        font-weight:800;
        border-radius:12px;
        background:#0C4A6E;
}</string>
   </property>
   <property name="text">
    <string>UPDATE LAST NAME</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update_qual_btn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
		font-family:Arial, sans-serif;
		font-size:18px;
        font-weight:800;
        border-radius:12px;
        background:#0C4A6E;
}</string>
   </property>
   <property name="text">
    <string>UPDATE QUALIFICATION</string>
   </property>
  </widget>
  <widget class="QPushButton" name="remove_stu_btn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>380</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
		font-family:Arial, sans-serif;
		font-size:18px;
        font-weight:800;
        border-radius:12px;
        background:#0C4A6E;
}</string>
   </property>
   <property name="text">
    <string>REMOVE STUDENT</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
